@xen-orchestra/lite/src/views/pool/PoolDashboardView.vue

```diff
@@ -17,11 +17,64 @@
 </template>
 
 <script lang="ts" setup>
+import { differenceBy } from "lodash-es";
+import { computed, onMounted, provide, readonly, watch } from "vue";
 import ProgressBar from "@/components/ProgressBar.vue";
+import PoolDashboardCpuUsage from "@/components/pool/dashboard/PoolDashboardCpuUsage.vue";
 import PoolDashboardStatus from "@/components/pool/dashboard/PoolDashboardStatus.vue";
 import UiBadge from "@/components/ui/UiBadge.vue";
 import UiCard from "@/components/ui/UiCard.vue";
 import UiTitle from "@/components/ui/UiTitle.vue";
+import useFetchStats from "@/composables/fetch-stats.composable";
+import { GRANULARITY, type HostStats, type VmStats } from "@/libs/xapi-stats";
+import type { XenApiHost, XenApiVm } from "@/libs/xen-api";
+import { useHostStore } from "@/stores/host.store";
+import { useVmStore } from "@/stores/vm.store";
+
+const hostStore = useHostStore();
+const vmStore = useVmStore();
+
+const {
+  register: hostRegister,
+  unregister: hostUnregister,
+  stats: hostStats,
+} = useFetchStats<XenApiHost, HostStats>("host", GRANULARITY.Seconds);
+const {
+  register: vmRegister,
+  unregister: vmUnregister,
+  stats: vmStats,
+} = useFetchStats<XenApiVm, VmStats>("vm", GRANULARITY.Seconds);
+
+const runningVms = computed(() =>
+  vmStore.allRecords.filter((vm) => vm.power_state === "Running")
+);
+
+provide("hostStats", readonly(hostStats));
+provide("vmStats", readonly(vmStats));
```

Comment:
To avoid forwarding the prop on several levels (eg: `PoolDashboardCpuUsage` will need to forward the prop to his sub component, `HostCpuUsage` and `VmCpuUsage` ) and to give access to the prop only to those components that need it. (eg: `PoolDashboardCpuUsage` don't really need or "consume" the stats.